.github/workflows/make.yml

```diff
@@ -0,0 +1,27 @@
+---
+name: Make
+'on':
+  pull_request:
+  push:
+    branches:
+      - master
+  schedule:
+    - cron: '0 0 * * *'
```

Comment:
There's no need for having a cron check here... I think it's fine on pushes and cron.

Reply:
Fixed in https://github.com/micheleg/dash-to-dock/pull/1657/commits/f219115cde96a1743d73d3aa8da7378afe7f2216